elmslie-plugin/src/main/resources/templates/PresentationActor.kt.ftl

```diff
@@ -0,0 +1,17 @@
+package ${presentation_package_name}
+
+import vivid.money.elmslie.core.store.Actor
+import ${presentation_package_name}.${domain_name}Command.*
+import ${presentation_package_name}.${domain_name}Event.Internal
+import io.reactivex.Observable
+import javax.inject.Inject
+
+internal class ${domain_name}Actor @Inject constructor(
+    // your dependencies
+) : Actor<${domain_name}Command, ${domain_name}Event> {
+
+    override fun execute(command: ${domain_name}Command): Observable<${domain_name}Event> =
+        when (command) {
```

Comment:
Is this line break necessary? It reduces the space for function implementation.
To keep the first line short it could be transformed to: 
```
override fun execute(
  command: ${domain_name}Command
): Observable<${domain_name}Event> = when(command)
```
or possibly to 
```
override fun execute(command: ${domain_name}Command) = when (command)
```